### ISSUE TITLE
Status of this project?

### ISSUE THREAD

**Issue by guettli** [zalando/PGObserver#109]
According to the code frequency it looks that this project does not get developed any more: 
   https://github.com/zalando/PGObserver/graphs/code-frequency

Please tell the users the state of the project in the README.

Is there are new project which I should use instead of this project?

**Comment by Jan-M** [comment]
Nope, while it is still heavily in use internally by those projects relying on stored procedures there is no new development happening.

Most teams have left stored procedures behind, only in specific uses cases and optimizations they find use now.

For our data and metrics tracking we rely on ZMON (zmon.io) today. And if you are not relying on versioned API Schemas you can basically use any tool to track per stored procedure metrics.

**Comment by a1exsh** [comment]
@guettli pgwatch2 might be worth a look: https://github.com/cybertec-postgresql/pgwatch2  As far as I understand it's a next iteration on the same idea, from someone heavily involved in working on PGObserver in the past. ;)
